### PR TITLE
Working with payloads

### PR DESCRIPTION
**Pull request by nicolasparrague**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The lookup component provides the following features:
 
 2. **Create the `LookupDefinition` record**
    
-   Go to Setup > Custom Metadata Types > LookupDefinition > add a new record
+   Go to Setup > Custom Metadata Types > onclick in  Manage Records (LookupDefinition) > add a new record
```
Comment (SantiagoParraGuerrero) [inline README.md:66]: onclick is not a word

File: README.md [hunk 2]
```diff
@@ -143,6 +143,7 @@ public List<LookupResult> getDefaultSearchResults(Map<String, Object> payload) {
     return result;
 }
 ```
+**IMPORTANT**: `selectedIds` *is refers to Id selected must be used in `getSelection()` method and `getSearchResults()` method*
```
Comment (SantiagoParraGuerrero) [inline README.md:146]: check you english in a translator before putting it in the documentation
also selectedIds does not need to be used but it is recommended

File: README.md [hunk 3]
```diff
@@ -170,7 +171,8 @@ singleLookupResult.recordPayload = new Map<String, Object> {
 ```js
 handleChange(event) {
     const value = event.detail.value;// ["someId", "someId2", "someId3"]
-    const payload = event.detail.payload;// {"someId": {type: nurse, name : johana}, someId2: {ty...}}
+    const payload = { someId : event.detail.value };// {"someId": {type: nurse, name : johana}, someId2: {ty...}}
+    // Overwrite the component with the Id of the payload to consult
```
Comment (SantiagoParraGuerrero) [inline README.md:175]: I dont understand this comment

File: src-lookup-sample/main/default/classes/AccountContactLookup.cls [hunk 4]
```diff
@@ -1,100 +1,98 @@
...
-        'iconName' => 'standard:contact'
-    };
-
-    public List<LookupResult> getSelection(
-        List<String> selectedIds,
-        Map<String, Object> payload) {
-
-        String accountId = (String) payload.get('accountId');
-        List<LookupResult> result = new List<LookupResult>();
+  public final static Map<String, Object> CONTACT_ICON = new Map<String, Object>{
```
Comment (SantiagoParraGuerrero) [inline src-lookup-sample/main/default/classes/AccountContactLookup.cls:3]: why does it use the contact icon if this is a Contact SObject Lookup?

**Review by SantiagoParraGuerrero — APPROVED** [review]

File: src-lookup-sample/main/default/classes/AccountContactLookup.cls (hunk 4 above)
Comment (nicolasparrague) [reply src-lookup-sample/main/default/classes/AccountContactLookup.cls:3]: I don't understand this comment, if the lookup is responsible for showing contacts, shouldn't it be contact icons?

**Pull request merged**